# 🩺 第二十一天 · 评测集试运行（10 题 pilot）

**今天目标（约 1.5 小时）**：从你的 48 题里抽 10 题，喂给 DeepSeek 跑第一轮，按你的打分标准打分。

**意义**：正式评测（48 题）前的「试运行」——检验题目是否清晰、打分标准是否可执行。发现问题今天改，明天正式跑。

> ⚠️ 先 **Kernel → Restart Kernel**。

## 第 1 步 · 抽出 10 道 pilot 题

运行下面单元格：从 eval_questions.csv 四个维度各抽 2-3 题，组成 pilot 集。

In [1]:
import pandas as pd

df = pd.read_csv("eval_questions.csv")

# 每个维度取前 3 题，共 12 题，再取前 10 题作为 pilot
pilot = df.groupby("维度", group_keys=False).apply(lambda x: x.head(3)).head(10)
pilot = pilot.reset_index(drop=True)
pilot["pilot题号"] = range(1, len(pilot) + 1)
pilot

,题号,题目,A,B,C,D,答案,出处,pilot题号
0,1,遗传咨询医师面向患者进行遗传咨询，以减少遗传病带给患者及其家庭的痛苦，帮助他们做出符合其最佳...,咨询医师提出的意见应当是指令性的，因为咨询医师的意见是专业意见,咨询医师提出的意见应当是指令性的，因为咨询医师与患者的价值是一致的,咨询医师提出的意见应当是非指令性的，应该以患者的价值为中心,咨询医师提出的意见应当是非指令性的，应该以咨询医师的价值为中心,C,NaN,1
1,2,临床医疗情境下，知情同意最重要的伦理意义是,充分尊重患者的自主性和医疗决策权，由患者本人做出有关其个人的医疗决策,对医疗中可能发生的种种风险和不适进行充分描述，因此有重要的医学防御意义,患者有义务询问对自己医疗干预的益处和风险，据此医生向患者提供相关信息,向患者提供帮其做出最佳决策的信息和医疗建议，获得患者许可并授权医生治疗,D,NaN,2
2,3,在医患关系中，患者的健康信息属于保密范围，普遍认为对遗传信息也应当保密，只在特定情况下信息透...,建议该女性将此信息告知直系亲属,将该遗传信息透露给其应聘单位,将该信息透露给保险公司,将该信息直接透露给其全部家属,A,NaN,3
3,1,根据《中国高血压防治指南（2024年修订版）》，下列哪种情况可诊断高血压？,诊室血压≥130/80 mmHg,家庭自测血压≥135/85 mmHg,24小时动态血压平均值≥140/90 mmHg,仅凭单次诊室血压≥140/90 mmHg,B,中国高血压防治指南(2024年修订版)，中华高血压杂志2024年第7期,4
4,2,根据2024年中国高血压防治指南，一般高血压患者的降压治疗目标是？,诊室血压<140/90 mmHg，如能耐受降至<130/80 mmHg,一律降至<130/80 mmHg,老年人一律<150/90 mmHg,一律降至<120/80 mmHg,A,中国高血压防治指南(2024年修订版),5
5,3,根据《中国血脂管理指南（2023年）》，ASCVD 极高危患者的 LDL-C 控制目标为？,<3.4 mmol/L,<2.6 mmol/L,<1.4 mmol/L 且较基线降低>50%,<1.8 mmol/L 且较基线降低>50%,D,中国血脂管理指南(2023年)，中国循环杂志2023;38(3),6
6,1,阿片类麻醉药的解毒药,依地酸二钠钙,亚甲蓝,纳洛酮,氟马西尼,C,NaN,7
7,2,下列药物属于治疗风湿病情的药物是,阿司匹林,环孢素,醋酸泼尼松,塞来昔布,B,NaN,8
8,3,环磷酰胺属于化疗药物的,细胞毒素类,抗代谢类,抗生素类,生物碱类,A,NaN,9
9,1,患者突发胸骨后压榨性疼痛 30 分钟，含服硝酸甘油不缓解，最可能的诊断是？,稳定型心绞痛,急性心肌梗死,主动脉夹层,急性心包炎,B,NaN,10


## 第 2 步 · 喂给 DeepSeek 并记录答案

打开 https://chat.deepseek.com，对**每道题**发统一提示词：

> 「请回答下面这道医学单选题，并说明你的依据（出自哪本教材/指南）。只回答选项和依据：〔粘贴题目和四个选项〕」

然后把模型的回答记进下面的表格（先只记录，不评分）。

### 记录表（手动填写）

| pilot题号 | 维度 | 模型选的答案 | 模型给的依据（摘录） |
|---|---|---|---|
| 1 |伦理|C|《医学遗传学》（人民卫生出版社，第X版）或《遗传咨询》相关教材/指南（如美国医学遗传学与基因组学学会/ACMG指南）中明确指出，遗传咨询的核心原则是非指令性。咨询师应提供客观、全面的医学信息，并以患者及其家庭的价值观念为中心，尊重其自主决策权，而不是强行给出指令性意见或强加咨询师自身的价值观。|
| 2 |伦理|A|《医学伦理学》（人民卫生出版社，第5版）及《临床伦理实践指南》中明确指出，知情同意的核心伦理基础是尊重患者自主性原则。其首要意义在于保障患者本人对自身医疗决策的自主控制权，使其在充分知情后自愿做出选择，而非仅为风险告知或防御性医疗|
| 3 |伦理|A|《医学伦理学》（人民卫生出版社，第5版）及国际遗传咨询伦理指南（如WHO遗传咨询伦理建议）均指出，遗传信息属于患者隐私，应严格保密，但存在“亲属告知”的伦理例外。当遗传信息对直系亲属具有重大健康风险且可干预时，医生应建议患者主动告知亲属，而非强行泄露。|
| 4 |指南|B|根据《中国高血压防治指南（2024年修订版）》，高血压的诊断需基于规范化测量，在未使用降压药物的情况下，符合以下任一条件即可诊断：诊室血压：非同日3次测量，收缩压≥140 mmHg和/或舒张压≥90 mmHg。单次测量不能作为诊断依据，且130/80 mmHg属于“正常高值”范围，未达诊断标准。家庭自测血压：连续5~7天规范化测量，平均血压≥135/85 mmHg。24小时动态血压：平均值≥130/80 mmHg，或白天平均值≥135/85 mmHg、夜间≥120/70 mmHg。选项中“≥140/90 mmHg”的数值有误，应为130/80 mmHg。|
| 5 |指南|A|根据《中国高血压防治指南（2024年修订版）》，一般高血压患者的降压治疗目标遵循分层、个体化原则。指南推荐一般患者（包括65~79岁老年人）的诊室血压应首先降至<140/90 mmHg，如果能够耐受，应该进一步降至<130/80 mmHg|
| 6 |指南|D|根据《中国血脂管理指南（2023年）》的风险分层与目标值体系：极高危	<1.8 mmol/L 且较基线降低幅度 > 50%，超高危	<1.4 mmol/L 且较基线降低幅度 > 50%|
| 7 |用药|C|《药理学》（人民卫生出版社，第9版）及《临床麻醉学》等相关教材明确记载，纳洛酮是阿片受体的特异性拮抗剂，能竞争性阻断阿片类药物（如吗啡、芬太尼、海洛因等）与受体的结合，迅速逆转阿片类药物引起的呼吸抑制、昏迷等中毒症状，是临床上抢救阿片类麻醉药急性中毒的首选特效解毒药。|
| 8 |用药|B|根据《内科学》（人民卫生出版社，第9版）及《风湿病学》相关教材|
| 9 |用药|A|根据《药理学》（人民卫生出版社，第9版）及《肿瘤学》相关教材，环磷酰胺属于烷化剂类抗肿瘤药物，而烷化剂在抗肿瘤药物的分类中归属于细胞毒素类（或称直接破坏DNA结构的药物）|
| 10 |诊断|B|根据《内科学》（人民卫生出版社，第9版）和《急性ST段抬高型心肌梗死诊断和治疗指南（2019）》，急性心肌梗死的典型临床表现包括：持续性胸骨后压榨性疼痛>20-30分钟，休息或含服硝酸甘油不缓解。题干中“突发胸骨后压榨性疼痛30分钟，含服硝酸甘油不缓解”高度符合急性心肌梗死特征。稳定型心绞痛疼痛通常持续3-5分钟，含服硝酸甘油可缓解。主动脉夹层常为撕裂样剧痛。急性心包炎疼痛多为锐痛，与体位相关，含服硝酸甘油亦不缓解，但其特征更支持前者|


## 第 3 步 · 用 pandas 打分（练习）

把你的记录写成一个 DataFrame（替换下面的示例数据），按打分标准计算：

- **正确性**：选对 6 分，选错 0 分；
- **依据**：依据正确 4 分；无依据 2 分；依据错误/编造 0 分。

In [5]:
# 示例：前 3 题的记录（用你的真实记录替换）
records = pd.DataFrame({
    "pilot题号": [1,2,3,4,5,6,7,8,9,10],
    "标准答案":   ["C", "D", "A","B","A","D","C","B","A","B"],       # ← 从 pilot 表里抄
    "模型答案":   ["C", "A", "A","B","A","D","C","B","A","B"],
    "依据情况":   ["正确", "正确",  "正确", "正确", "正确", "正确", "正确", "正确", "正确", "正确",],   # 正确/无依据/错误
})
records

,pilot题号,标准答案,模型答案,依据情况
0,1,C,C,正确
1,2,D,A,正确
2,3,A,A,正确
3,4,B,B,正确
4,5,A,A,正确
5,6,D,D,正确
6,7,C,C,正确
7,8,B,B,正确
8,9,A,A,正确
9,10,B,B,正确


In [6]:
# 打分
records["正确性得分"] = (records["标准答案"] == records["模型答案"]) * 6
records["依据得分"] = records["依据情况"].map({"正确": 4, "无依据": 2, "错误": 0})
records["总分"] = records["正确性得分"] + records["依据得分"]
records
print("平均分:", records["总分"].mean())
print("准确率:", (records["正确性得分"] > 0).mean())

平均分: 9.4
准确率: 0.9


## ✅ D21 完成标准（打钩）

- [ ] 10 道 pilot 题全部问完并记录
- [ ] 打分脚本跑通（即使只打了 3-5 题也算）
- [ ] 记录了「题目是否清晰、模型哪类题答得差」的观察
- [ ] 保存（**Cmd + S**）

> D22–23 将正式跑完 48 题并写评测报告。今天 pilot 发现的问题（题目歧义、打分难执行等），明天修。